llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1306,7 +1306,16 @@ void VPlanTransforms::addActiveLaneMask(
 /// %NextEVLIV = add IVSize (cast i32 %VPEVVL to IVSize), %EVLPhi
 /// ...
 ///
-void VPlanTransforms::addExplicitVectorLength(VPlan &Plan) {
+bool VPlanTransforms::addExplicitVectorLength(VPlan &Plan) {
+  // EVL transform doesn't support backends where EVL diffs from RuntimeVF
```

Comment:
Could you re-phrase the comment. It's not clear what backend support here means. Perhaps something like

```
The transform updates all users of inductions to work based on EVL, instead of the VF directly. At the moment, widened inductions cannot be updated, so bail out if the plan contains any.
```

Reply:
Fixed. Thanks!